tdp_vars_defaults/exporter/exporter.yml

```diff
@@ -25,3 +25,33 @@ node_exporter_web_listen_address: --web.listen-address=:{{ exporter_node_http_po
 
 # node exporter service start on boot policies
 node_exporter_start_on_boot: false
+
+#####################
+# postgres-exporter #
+#####################
+# postgres-exporter users and group
+postgres_exporter_user: postgres-exporter
+postgres_exporter_group: postgres-exporter
+
+# postgres exporter version
+postgres_exporter_release: postgres_exporter-0.12.0.linux-amd64
```

Comment:
This should be added on tdp-getting-started [tdp-release-uri](https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/tdp-release-uris.txt)

Reply:
Added the PR https://github.com/TOSIT-IO/tdp-getting-started/pull/262